projects/court-case-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/proxy/CommunityApiController.kt

```diff
@@ -202,18 +202,27 @@ class CommunityApiController(
                 .map(CompletableFuture<CompareReport>::join)
                 .collect(Collectors.toList())
         }
+
+        val executed = reports.filter { it.testExecuted == true }
+        val successful = reports.filter { it.success && it.testExecuted == true }
         val unsuccessful = reports.filter { !it.success && it.testExecuted == true }
```

Comment:
Minor - but there's another handy function for that in Kotlin 😄 
```suggestion
        val (successful, unsuccessful) = executed.partition { it.success }
```